include/picongpu/param/incidentField.param

```diff
@@ -32,6 +32,7 @@
  *  - profiles::PulseFrontTilt<>      : Gaussian beam with tilted pulse front with given parameters
  *  - profiles::Wavepacket<>          : wavepacket with Gaussian spatial and temporal envelope profile with given
  * parameters
+ *  - profiles::DispersiveLaser<>     : dispersive Gaussian Beam with possibilities to set SD, AD, GDD and TOD

```

Comment:
Could it be that comment for the spectral laser profile is missing?

Reply:
The SpectralLaser profile should not be part of this PR anyways. It is just a (unnecessary) specialization of the DispersiveLaser.